tools/templates/Makefile-common-macros.in

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2015 The Perl Foundation
+
+.NOTPARALLEL:
```

Comment:
This line needs to be removed. It's been added already in tools/templates/Makefile-common-rules.in. See https://github.com/rakudo/rakudo/commit/44e0386afd55a8d9afe273e6a872a4e80936eb37